_rules/device-motion-disabled-c249d5.md

```diff
@@ -0,0 +1,304 @@
+---
+id: c249d5
+name: Device motion can be disabled
+rule_type: atomic
+description: |
+  This rule checks that it is possible to disable functionality that can be operated by device motion.
+accessibility_requirements:
+  wcag21:2.5.4: # Motion Actuation (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Accessibility tree
+acknowledgments:
+  authors:
+    - João Vicente
+    - Carlos Duarte
+htmlHintIgnore:
+  # https://www.npmjs.com/package/htmlhint
+  # (used with `npm test` to ensure validity of code snippets)
+  - 'attr-lowercase'
+---
+
+## Applicability
+
+The rule applies to an [HTML document][] with an associated [Window object][] that has one or more registered [device orientation events][device orientation] or [device motion events][device motion].
```

Comment:
I like listing the events.

Is "registered" clear enough, or does that need further definition? I am not sure…
Maybe something like "has one or more EventTarget whose event listener list contains …"
https://dom.spec.whatwg.org/#eventtarget
Using DOM specs would also make the applicability easier to automate, which is always helpful for testers…

...hmmm wondering if such a formulation could be used to target the EventTarget (or even the event listener) rather than the full page (and thus give more precise result as we could pinpoint which element is not correct).

Reply:
Thanks for the suggestion. I also has some reserves about using registered. 
Given that we know the eventTarget to be the Window object, I'll focus instead on the event listener list.